docs/docs/cmd/util/accesstoken/accesstoken-get.mdx

```diff
@@ -30,6 +30,12 @@ The `util accesstoken get` command returns an access token for the specified res
 
 If the URL of your SharePoint tenant has been previously retrieved, you can use `sharepoint` as the resource to retrieve access token for SharePoint. To verify if the URL of your SharePoint tenant has been previously retrieved, use the [`spo get`](../../spo/spo-get.mdx) command. To explicitly set the URL of your SharePoint tenant use the [`spo set`](../../spo/spo-set.mdx) command.
 
+:::tip
+
+You can use `sharepoint` as the resource to retrieve access token for SharePoint and `graph` to retrieve access token for Microsoft Graph.
+
+:::
```

Comment:
We usually put our admonitions at the top of the section. Let's do that here as well.

Reply:
Didn't know that. Done.